### ISSUE TITLE
Saga serialization configuration ignores the custom serialize settings

### ISSUE THREAD

**Issue by mfahadi** [rebus-org/Rebus.PostgreSql#40]
```
s.UseSagaSerializer(new JsonSagaSerializer(new JsonSerializerSettings()
{
    TypeNameHandling = TypeNameHandling.None
}));
```

Above settings has no affect on serialization of sagas, I believe the culprit is below line and should be used like `c.Has<ISagaSerializer>(false)`, same as Sql Server implementation.

https://github.com/rebus-org/Rebus.PostgreSql/blob/b5b33b10a1c198cb09c2613ebe95a481453f22b4/Rebus.PostgreSql/Config/PostgreSqlConfigurationExtensions.cs#L68

[Link to the usuage in Sql Server repo](https://github.com/rebus-org/Rebus.SqlServer/blob/202fdd961136a4c7bf15c2ee1fd2684abbfb0603/Rebus.SqlServer/Config/SqlServerSagaConfigurationExtensions.cs#L34)

A related question, should not we use System.Text.Json serializer as default as it is used for transport.

**Comment by mookid8000** [comment]
Fixed by you @mfahadi in #42 - out in Rebus.PostgreSql 9.0.0-alpha05 on NuGet.org now 🙂 thanks again!

**Comment by mfahadi** [comment]
@mookid8000 thanks for the quick merge :)
